Formula/p11-kit.rb

```diff
@@ -39,11 +41,10 @@ def install
 
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
-                          "--disable-trust-module",
                           "--prefix=#{prefix}",
                           "--sysconfdir=#{etc}",
                           "--with-module-config=#{etc}/pkcs11/modules",
-                          "--without-libtasn1"
```

Comment:
Do we need to add the `libtasn1` dependency to enable the `trust-module`? (I'm guessing the answer is yes, but it would be good for it to on the record in this PR.)

Reply:
Sure, the [p11-kit compilation documentation](https://p11-glue.github.io/p11-glue/p11-kit/manual/devel-building.html#devel-building-configure) mentions "libtasn1 is required to build the trust module and code that interacts with certificates."